tests/b020.py

```diff
@@ -0,0 +1,17 @@
+items = [1, 2, 3]
```

Comment:
Lets comment on where the errors are to help future readers like other tests
```suggestion
"""
Should emit:
B020 - on lines 8 and 21
"""

items = [1, 2, 3]
```